packages/example-log-extension/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
```

Comment:
2017, 2018.

(All files -- since others in loopback-next were already updated).

Reply:
Let's use `slt license` because that's the tool we will use in the future.

```
SLT_OWNER="IBM Corp." slt copyright
```

Unfortunately this tool sets the copyright year to 2018 because it uses `git` commit history to find out the date of the first commit. Do you think this is a problem?